product_docs/docs/epas/15/reference/sql_reference/02_data_types/06_xml_type.mdx

```diff
@@ -80,36 +78,36 @@ XMLSERIALIZE ( { DOCUMENT | CONTENT } value AS type [ [ NO ] INDENT ] )
 
-Processing XML data with PostgreSQL will be less error-prone and more efficient if the XML data encoding, client encoding, and server encoding are the same. Since XML data is internally processed in UTF-8, computations will be most efficient if the server encoding is also UTF-8.
+Processing XML data with PostgreSQL is less error prone and more efficient if the XML data encoding, client encoding, and server encoding are the same. Since XML data is internally processed in UTF-8, computations are most efficient if the server encoding is also UTF-8.
 
 !!!Note
-Some XML-related functions may not work at all on non-ASCII data when the server encoding is not UTF-8. This is known to be an issue for `xmltable()` and `xpath()` in particular.
+Some XML-related functions might not work at all on non-ASCII data when the server encoding isn't UTF-8. This is known to be an issue for `xmltable()` and `xpath()` in particular.
 !!!
 
-## Accessing XML Values 
+## Accessing XML values 
 
-The XML data type is unusual because it does not provide any comparison operators. This is because there is no well-defined and universally useful comparison algorithm for XML data. One consequence of this is that you cannot retrieve rows by comparing an XML column against a search value. XML values should therefore typically be accompanied by a separate key field such as an ID. An alternative solution for comparing XML values is to convert them to character strings first, but note that character string comparison has little to do with a useful XML comparison method.
+The XML data type is unusual because it doesn't provide any comparison operators. This is because there's no well-defined and universally useful comparison algorithm for XML data. One consequence of this is that you can't retrieve rows by comparing an XML column against a search value. XML values are therefore typically accompanied by a separate key field such as an ID. An alternative solution for comparing XML values is to convert them to character strings first. However, character string comparison has little to do with a useful XML comparison method.
 
-Because there are no comparison operators for the XML data type, it is not possible to create an index directly on a column of this type. If speedy searches in XML data are desired, possible workarounds include casting the expression to a character string type and indexing that, or indexing an XPath expression. Of course, the actual query would have to be adjusted to search by the indexed expression.
+Because there are no comparison operators for the XML data type, you can't create an index directly on a column of this type. If you want speedy searches in XML data, possible workarounds include casting the expression to a character string type and indexing that, or indexing an XPath expression. Of course, you must adjust the actual to search by the indexed expression.
```

Comment:
"the actual" => "the actual query"

Reply:
```suggestion
Because there are no comparison operators for the XML data type, you can't create an index directly on a column of this type. If you want speedy searches in XML data, possible workarounds include casting the expression to a character string type and indexing that, or indexing an XPath expression. Of course, you must adjust the actual query to search by the indexed expression.
```